main/resource-access.bs

```diff
@@ -95,6 +95,44 @@ Behaviour pertaining to authorization MUST proceed this optional redirect
 
 ## Linked Data ## {#linked-data}
 
+### Storage ### {#storage}
+
+When a servers supports a data pod, it MUST provide one or more storages
+(`pim:Storage`). A storage is the root container for all of its contained
+resources (see [[#resource-containment]]).
+
+When a server supports multiple storages, the URIs MUST be allocated to
+non-overlapping space.
+
+Servers exposing the storage resource MUST advertise by including the HTTP
+`Link` header with `rel="type"` targeting
+`http://www.w3.org/ns/pim/space#Storage` when responding to request URI.
```

Comment:
What request URI? All? The storage? Root?

Reply:
The storage. https://github.com/solid/specification/pull/208/commits/4356daf06a0ca2216b9cbf76a856e2c82badc6c5